ci-operator/step-registry/ipi/conf/gcp/filestore-workload-identity-federation/ipi-conf-gcp-filestore-workload-identity-federation-ref.yaml

```diff
@@ -0,0 +1,16 @@
+ref:
+  as: ipi-conf-gcp-filestore-workload-identity-federation
+  from_image:
+    namespace: ocp
+    name: "4.12"
+    tag: upi-installer
```

Comment:
Shouldn't this be a newer version? Where does `upi-installer` come from?

Reply:
The step just needs to call gcp sdk, upi-installer image installed the sdk, but it does not care about the exact version of upi-installer, because all versions of upi-installer installed the gcp sdk.